tests/unit/amo/components/TestAddonReviewList.js

```diff
@@ -468,6 +469,19 @@ describe(__filename, () => {
       expect(img).toHaveProp('src', addon.icon_url);
     });
 
+    it('renders a class name with its type', () => {
+      const addon = {
+        ...fakeAddon,
+        type: ADDON_TYPE_STATIC_THEME,
+      };
+      dispatchAddon(addon);
```

Comment:
You could probably move the `addon` above into the `dispatchAddon()`, like the test case after this one.